charts/scalardb/Chart.yaml

```diff
@@ -16,8 +16,8 @@ sources:
   - https://github.com/scalar-labs/scalardb
 dependencies:
 - name: envoy
-  version: ~2.2.0
-  repository: https://scalar-labs.github.io/helm-charts
+  version: ~3.0.0-SNAPSHOT
+  repository: file://../envoy/
```

Comment:
Since we don't provide SNAPSHOT version charts in our Helm Repository for users, we have to specify the dependency by using a file path.